packages/esm-patient-chart-app/src/index.ts

```diff
@@ -159,7 +159,7 @@ export const genericDashboard = getSyncLifecycle(genericDashboardComponent, {
   moduleName,
 });
 
-// t('startAVisit', 'Start a visit')
+// t('startVisit', 'Start visit')
```

Comment:
Thanks @vasharma05 , however this will change the design, from `Start a visit` to `Start visit`.

Reply:
We should probably change that key to something different altogether. I propose `startVisitWorkspaceTitle` to match what we're doing in the [orders app](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-orders-app/src/index.ts#L21). Additionally, and maybe this is beyond the scope of this PR, we should also probably register this workspace using the `workspaces` key in the `routes.json` file. Registering workspaces using extensions isn't the ideal approach at this point.